.scalafmt.conf

```diff
@@ -0,0 +1,32 @@
+version = 3.7.17
```

Comment:
Don't add Scalafmt in to RocketChip in this commit(although I also like to do it)